release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1174,6 +1179,14 @@ With this release, replacement control plane nodes are assigned to the correct i
 
 * The previous resolution of link:https://issues.redhat.com/browse/OCPBUGS-5546[OCPBUGS-5546] removed the `clusterName` assignment of `MachineConfig.Name` in the machine configuration object. As a result, the value of the parameter was an empty string and, when it was combined with the value of `machineName` to create an IP address name, it created an invalid value. The invalid value caused machines to fail during provisioning. With this release, the value for `clusterName` is obtained from the infrastructure object so that it creates a valid IP address name. (link:https://issues.redhat.com/browse/OCPBUGS-7696[*OCPBUGS-7696*])
 
+* The Kubernetes 1.26 release introduced changes to the node infrastructure, such as removing an unhealthy node with an `NotReady` status from the public load balancer to prevent the node from receiving routing traffic. These changes can impact a {product-title} node that runs inside a cluster on Microsoft Azure, because removing a node from a public load balancer can result in the node losing internet connectivity. This issue might render the node unable to regain a ready status and establish an outbound connection. The {product-title} {product-version} release fixes this issue  so that a node marked with `NotReady` status is detected by the `kube-proxy` based health probes, now the default for services of `type=LoadBalancer` with `externalTrafficPolicy=Cluster`, without the need of node detachment from the public load balancer. This means that a node can retain an outbound internet connection throughout these phases. (link:https://issues.redhat.com/browse/OCPBUGS-7359[*OCPBUGS-7359*])
```

Comment:
```suggestion
* The Kubernetes 1.26 release introduced changes to the node infrastructure, such as removing an unhealthy node with an `NotReady` status from the public load balancer to prevent the node from receiving routing traffic. These changes impacted a node that ran inside a cluster on Microsoft Azure. As a result, the node was unable to regain a ready status and establish an outbound connection. With this update, a node marked with `NotReady` status is detected by the `kube-proxy` based health probes without the need of node detachment from the public load balancer. This means that a node can retain an outbound internet connection throughout these phases. (link:https://issues.redhat.com/browse/OCPBUGS-7359[*OCPBUGS-7359*])
```